### DISCUSSION TITLE
Certain apps cannot be used due to accessibility being turned on.

### DISCUSSION THREAD

**Discussion by Annswwr** [tribalfs/GalaxyMaxHzPub#147] [Q&A] [answered]
It seems like certain apps like banking apps cannot be used due to security concerns. Is there any way to bypass this limitation or disable accessibility only for certain apps?

![photo_2024-01-08_17-22-38](https://github.com/tribalfs/GalaxyMaxHzPub/assets/139935994/6e2637d6-0ff1-49dd-af3d-6d9b496960aa)

**Comment by tribalfs** [comment, marked as answer]
It's a matter of bank app's policy.  
The app can do nothing about it unfortunately. 
It can't automatically disable accessibility when opening certain apps as it needs accessibility service in order to do that.

**Reply by Annswwr** [reply]
That's unfortunate but completely understandable.
I've found that the only mods that don't require accessibility features are the 'Quick-doze' and 'Animation' mods, which are both great.
The main refresh mods seem to work just fine so it's not really a big deal.

Thanks for the help!

**Reply by Scarlaid** [reply]
that's a bummer
any alternative way to not use accessibility and still forcefully change AOD to 60hz?

**Comment by tribalfs** [comment]
Update:

Starting 8.1.90, you can use the added master switch to temporarily disable GMH service.
